### ISSUE TITLE
Buttons don't have id

### ISSUE THREAD

**Issue by yahirah** [yahirah/Responsum-Test-Framework#5]
None of the buttons have id assigned.

**Comment by rostgaard** [comment]
You should be able to locate a button using a query on context (section id) node types and classes (widget class and then button class).
If there is a specific request to add ID's, you will probably have better luck pinging @ThomasLocke via an issue in https://github.com/ThomasLocke/ReceptionistClientFlex

**Comment by yahirah** [comment]
Yep, I'm totally able to locate it in diferent way, I just can see that most of the not-nested, not-autogenerated items in the GUI has an ID, which is pretty convenient and makes code much easier to maintenance. This issue is more like a note, so I don't forget it and can send Thomas a list of requests at one point in future, if it persists to a new version.

**Comment by ThomasLocke** [comment]
I'm a big fan of id's in HTML, as they remove all ambiguity from the source, which I personally find very nice.

For various reasons we don't do id's on significant elements in the new UI, but we can of course change that.

I know that @rostgaard is not a big fan, so I wont start adding them willy-nilly, unless he's on board with the idea.
